covasim/population.py

```diff
@@ -28,6 +28,14 @@ class People(list):
     people with a certain attribute).
     '''
 
+    def __init__(self, people):
+        self.array = np.array(people)
+
+
+    def get_list(self, indices):
+        return self.array[indices]
```

Comment:
This returns an array, not a list, which seems intentional, but could be confusing.